assets/airports/kcvg.json

```diff
@@ -0,0 +1,6856 @@
+{
+    "radio": {
+        "twr": "Cincinnati Tower",
+        "app": "Cincinnati Approach",
+        "dep": "Cincinnati Departure"
+    },
+    "icao": "KCVG",
+    "iata": "CVG",
+    "magnetic_north": -4,
+    "ctr_ceiling": 20000,
+    "initial_alt": 10000,
+    "position": ["N39d02.93", "W084d40.07", "896ft"],
+    "rr_radius_nm": 5,
+    "rr_center": ["N39d00.96", "W84d42.20"],
+    "has_terrain": false,
+    "wind": {
+        "angle": 180,
+        "speed": 10
+    },
+    "airspace": [
+        {
+            "floor": 0,
+            "ceiling": 200,
+            "airspace_class": "B",
+            "poly": [
+                ["N039d34m41.256","W085d19m02.839"],
+                ["N039d27m20.377","W085d30m00.935"],
+                ["N038d57m29.947","W085d30m00.144"],
+                ["N038d54m59.996","W085d27m42.091"],
+                ["N038d40m06.213","W085d27m44.092"],
+                ["N038d28m24.058","W084d57m59.920"],
+                ["N038d26m13.100","W084d01m05.851"],
+                ["N038d54m36.277","W083d54m42.777"],
+                ["N038d54m03.963","W083d49m47.047"],
+                ["N038d55m06.921","W083d54m42.034"],
+                ["N039d10m45.153","W083d52m59.870"],
+                ["N039d28m44.291","W083d58m01.005"],
+                ["N039d32m20.129","W084d17m44.127"],
+                ["N039d35m00.172","W084d43m29.965"]
+            ]
+        }
+    ],
+    "fixes": {
+        "APE": ["N040d09m03.826", "W082d35m17.885"],
+        "AXC": ["N039d44m15.077", "W088d51m23.153"],
+        "BKW": ["N037d46m49.079", "W081d07m24.440"],
+        "BVT": ["N040d33m22.218", "W087d04m09.748"],
+        "BWG": ["N036d55m43.467", "W086d26m36.363"],
+        "CTW": ["N040d13m45.151", "W081d28m35.316"],
+        "CVG": ["N039d00m57.532", "W084d42m12.045"],
+        "DJB": ["N041d21m29.034", "W082d09m43.087"],
+        "DQN": ["N040d00m59.200", "W084d23m48.833"],
+        "ENL": ["N038d25m11.995", "W089d09m32.392"],
+        "FLM": ["N038d38m57.990", "W084d18m38.166"],
+        "HNN": ["N038d45m14.849", "W082d01m34.203"],
+        "HVQ": ["N038d20m58.840", "W081d46m11.684"],
+        "HYK": ["N037d57m58.856", "W084d28m21.061"],
+        "IIU": ["N038d06m12.471", "W085d34m38.770"],
+        "JOT": ["N041d32m47.103", "W088d19m06.279"],
+        "MIE": ["N040d14m14.253", "W085d23m38.532"],
+        "PXV": ["N037d55m41.954", "W087d45m44.564"],
+        "ROD": ["N040d17m16.083", "W084d02m35.147"],
+        "SHB": ["N039d37m57.153", "W085d49m27.539"],
+        "TTH": ["N039d29m20.190", "W087d14m56.447"],
+        "TVT": ["N040d27m29.037", "W082d07m36.524"],
+        "VHP": ["N039d48m53.025", "W086d22m02.999"],
+        "VXV": ["N035d54m17.419", "W083d53m41.015"],
+        "YRK": ["N038d38m38.879", "W082d58m42.022"],
+      "AHTIY": ["N039d54m51.850", "W080d23m05.230"],
```

Comment:
I know you're separating fixes from VOR's here, but I think it's better to keep them all together in alphabetical order.